src/main/java/org/candlepin/config/ConfigProperties.java

```diff
@@ -129,6 +129,8 @@ private ConfigProperties() {
     // Space separated list of resources to hide in GET /status
     public static final String HIDDEN_CAPABILITIES = "candlepin.hidden_capabilities";
 
+    public static final String DB_HALT_STARTUP = "candlepin.db.halt_startup";
```

Comment:
"halt_startup" to me implies that we're trying to prevent something DB related from running its startup routine(s). If we're going to continue on the path of still running the checks every time, but not failing on error, we should rename this to something more intuitive, such as "halt_on_error", "allow_errors", or "allow_liquibase_desync".

Reply:
I think we only want a config option to skip the entire check (instead of not failing if it errors), so that kind of option would need a different name such as `candlepin.check_db_updated`

Reply:
I was going for "more information is better", but you are correct as they did not request this.
I reserve the right to bring this back up if it gets asked for :)